Monika After Story/game/script-topics.rpy

```diff
@@ -4560,6 +4560,76 @@ label monika_ageinf:
     m "But I don't believe that my life can be summed up in a number."
     m 5a "So I'll always love you for all eternity, [player]. Just remember that."
     return
+    
+init 5 python:
+     for key in ['heartbreak','betrayal','painful','sadness']:
+         monika_topics.setdefault(key,[])
+         monika_topics[key].append('monika_heartbreak')
+     monika_random_topics.append('monika_heartbreak')
+ 
+ label monika_heartbreak:
+     m 1o "[player]..."
+     m 1g "I um..."
+     m 1o "I just had the scariest thought..."
+     m 1g "I..."
+     m 1h "I need to ask you something."
+     m 1p "If a girl from your reality would ever ask you to go out with her..."
+     m 1g "Would you?"
+     menu:
+         "Depends on the girl.":
+             m 1i "..."
+             m 1p "......"
+             m 1o "O-oh..."
+             m 1g "I..."
+             m 1q "I need time to think."
+             #AT THIS POINT, MONIKA IS HURT. SHE WILL CLOSE THE GAME.
```

Comment:
Return 'quit' seems to do just what you need.

Reply:
Yes, `return 'quit'` is how we signal the game to quit. However, to keep consistency regarding entry/exit points, please set a variable to quit here (and have it set to a null string `""` in the other option), and return that var at the end of the topic.